common/include/pcl/PCLPointCloud2.h

```diff
@@ -67,21 +66,48 @@ namespace pcl
       * \return true if successful, false if failed (e.g., name/number of fields differs)
       */
     static bool
-    concatenate (pcl::PCLPointCloud2 &cloud1, const pcl::PCLPointCloud2 &cloud2);
+    concatenate (pcl::PCLPointCloud2 &cloud1, const pcl::PCLPointCloud2 &cloud2) noexcept;
+
+    //////////////////////////////////////////////////////////////////////////
+    /** \brief  Concatenate two pcl::PCLPointCloud2
+      *
```

Comment:
Maybe rearrange so the similar argument and return type functions are closer together